src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -88,13 +90,13 @@ const styles = StyleSheet.create({
     textAlign: 'left',
     color: isHeaderHide ? COLORS.white : COLORS.blackgrey
   }),
-  seeMemberText: {
+  seeMemberText: (followType) => ({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[500],
     textAlign: 'left',
-    color: COLORS.blue,
+    color: followType === 'incognito' ? COLORS.anon_secondary : COLORS.blue,
     marginTop: normalize(1)
-  }
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `seeMemberText` style now depends on the `followType` prop. This is a good use of dynamic styling based on component props.
```diff
- color: followType === 'incognito' ? COLORS.anon_secondary : COLORS.blue,
+ color: followType === 'incognito' ? COLORS.anon_secondary : (followType === 'signed' ? COLORS.blue : COLORS.default),
```
This change ensures that a default color is used when `followType` is neither 'incognito' nor 'signed'.


<!-- This is an auto-generated comment by OSS CodeRabbit -->